### PR TITLE
included instance type in ec2 instance list report.

### PR DESCRIPTION
**Pull request by amavinag**
Purpose or Intent
=================
  * included instance type in ec2 instance list report.

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 685d3545f32287cfb7bcf8c830b918e6891565c2:

Everything seems all right from lint perspective :smile: .

Commit inspection report:

**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: scripts/ec2cleanup.py [hunk 1]
```diff
@@ -46,10 +46,12 @@ def delete_old_instances(texts, ec2provider, provider_key, date,
...
-                print("EC2:{}  {}  \t {}".format(
-                    provider_key, instance, (date - creation)))
-                report.write("EC2:{}  {}  \t {}\n".format(
-                    provider_key, instance, (date - creation)))
+                print("EC2:{}  {}  \t {} \t {} \t {}".format(
+                    provider_key, instance, (date - creation),
+                    ec2provider.vm_type(instance), ec2provider.vm_status(instance)))
+                report.write("EC2:{}  {}  \t {} \t {} \t {}\n".format(
+                    provider_key, instance, (date - creation),
+                    ec2provider.vm_type(instance), ec2provider.vm_status(instance)))
```
Comment (psav) [inline scripts/ec2cleanup.py]: a) I'm not sure but ```ec2provider.vm_type``` and ```ec2provider.vm_status``` could be expensive
b) the message is the same in both cases, so why not.....

```python
                message = "EC2:{}  {}  \t {} \t {} \t {}".format(
                    provider_key, instance, (date - creation),
                    ec2provider.vm_type(instance), ec2provider.vm_status(instance))
                print(message)
                report.write(message)
```

Comment (amavinag) [inline scripts/ec2cleanup.py]: @psav will make changes for #b.

**Comment by amavinag** [comment]
@psav Thanks for the comment. Changes updated.

**Comment by dajoRH** [comment]
Lint report for commit 568b3baff9c28042e4d19e234471a931b5d3e4ac:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

File: scripts/ec2cleanup.py [hunk 2]
```diff
@@ -46,10 +46,11 @@ def delete_old_instances(texts, ec2provider, provider_key, date,
             report.write("\n{}:\n-----------------------\n".format(provider_key))
             for instance in ec2provider.list_vm(include_terminated=True):
                 creation = ec2provider.vm_creation_time(instance)
-                print("EC2:{}  {}  \t {}".format(
-                    provider_key, instance, (date - creation)))
-                report.write("EC2:{}  {}  \t {}\n".format(
-                    provider_key, instance, (date - creation)))
+                message = "EC2:{}  {}  \t {} \t {} \t {}\n".format(
```
Comment (mshriver) [inline scripts/ec2cleanup.py]: Consider labeling these fields, might become bothersome to parse the output/log with just a block of numbers/strings

Comment (amavinag) [inline scripts/ec2cleanup.py]: @mshriver makes sense.

**Comment by dajoRH** [comment]
Lint report for commit 0136b426fd50420e9943468ac9b3bb1d7559f76d:

Lint OK! :cake: :punch: :cookie: .


**No commit flaws detected**.
<mergeable/>

*CFME QE Bot*

**Pull request merged**